.fixtures.yml

```diff
@@ -4,24 +4,22 @@ fixtures:
     auditd: https://github.com/simp/pupmod-simp-auditd
     augeasproviders_core: https://github.com/simp/augeasproviders_core
     augeasproviders_grub: https://github.com/simp/augeasproviders_grub
-    augeasproviders_ssh: https://github.com/simp/augeasproviders_ssh
     concat:
       # master is beyond 4.1.1, but has breaking changes to
       # how fragments are ordered (MODULES-6625)
       repo: https://github.com/simp/puppetlabs-concat
       ref: 4.1.1
-    logrotate: https://github.com/simp/pupmod-simp-logrotate
+    pki: https://github.com/simp/pupmod-simp-pki
+    simplib: https://github.com/simp/pupmod-simp-simplib
+    stdlib: https://github.com/simp/puppetlabs-stdlib
+
+    # Fixtures for the AD acceptance test
```

Comment:
I'm not sure I see the benefit of separating out the fixtures based on the test that uses them.

Reply:
There's no good reason, I just thought I was separating it by unit vs acceptance so why not do it by suite.